<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFind_By">
     <property name="title">
      <string>Find By...</string>
     </property>
     <addaction name="actionTelephone_Number"/>
     <addaction name="actionCity"/>
    </widget>
    <addaction name="actionShow_All"/>
    <addaction name="menuFind_By"/>
    <addaction name="actionDelete_Phone_Number"/>
    <addaction name="actionChange_Call"/>
    <addaction name="actionAdd_Call"/>
    <addaction name="actionSort"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionTelephone_Number">
   <property name="text">
    <string>Telephone Number</string>
   </property>
  </action>
  <action name="actionCity">
   <property name="text">
    <string>City</string>
   </property>
  </action>
  <action name="actionDelete_Phone_Number">
   <property name="text">
    <string>Delete Phone Number</string>
   </property>
  </action>
  <action name="actionChange_Call">
   <property name="text">
    <string>Change Call</string>
   </property>
  </action>
  <action name="actionAdd_Call">
   <property name="text">
    <string>Add Call</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionCity</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFindBy_City_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Phone_Number</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDelete_Number_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpen_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onShow_All_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTelephone_Number</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFindBy_Telephone_Number_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Call</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddCall_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_Call</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onChangeCall_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSort</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSort_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShow_All_clicked()</slot>
  <slot>onFindBy_Telephone_Number_clicked()</slot>
  <slot>onFindBy_City_clicked()</slot>
  <slot>onDelete_Number_clicked()</slot>
  <slot>onOpen_clicked()</slot>
  <slot>onAddCall_clicked()</slot>
  <slot>onChangeCall_clicked()</slot>
  <slot>onSort_clicked()</slot>
 </slots>
</ui>
